packages/tasks/src/tasks/text-generation/spec/input.json

```diff
@@ -1,94 +1,204 @@
 {
 	"$id": "/inference/schemas/text-generation/input.json",
 	"$schema": "http://json-schema.org/draft-06/schema#",
-	"description": "Inputs for Text Generation inference",
+	"description": "Text Generation Input",
 	"title": "TextGenerationInput",
 	"type": "object",
+	"required": ["inputs"],
 	"properties": {
 		"inputs": {
-			"description": "The text to initialize generation with",
-			"type": "string"
+			"type": "string",
+			"example": "My name is Olivier and I"
 		},
 		"parameters": {
-			"description": "Additional inference parameters",
-			"$ref": "#/$defs/TextGenerationParameters"
-		},
-		"stream": {
-			"description": "Whether to stream output tokens",
-			"type": "boolean"
+			"$ref": "#/$defs/TextGenerationInputGenerateParameters"
 		}
 	},
 	"$defs": {
-		"TextGenerationParameters": {
-			"title": "TextGenerationParameters",
-			"description": "Additional inference parameters for Text Generation",
+		"TextGenerationInput": {
+			"type": "object",
+			"required": ["inputs"],
+			"properties": {
+				"inputs": {
+					"type": "string",
+					"example": "My name is Olivier and I"
+				},
+				"parameters": {
+					"$ref": "#/$defs/TextGenerationInputGenerateParameters"
+				}
+			}
+		},
```

Comment:
This is a re-declaration of the "root type" it seems - should not be needed right?

Reply:
Fixed in https://github.com/huggingface/huggingface.js/pull/629/commits/61b9d4e67f948239d3c4832ccabee866e2b67f91.